engine/flink/management/periodic/src/test/scala/pl/touk/nussknacker/engine/management/periodic/PeriodicDeploymentManagerTest.scala

```diff
@@ -123,14 +122,6 @@ class PeriodicDeploymentManagerTest extends FunSuite
     state.value.allowedActions shouldBe List(ProcessActionType.Cancel)
   }
 
-  test("deploy - should fail for invalid periodic property") {
```

Comment:
Because Perodic Deployment Manager doesn't take raw string now so it is impossible to pass invalid scenario. In this test was passed `GraphProcess("{}")`

Reply:
It is possible. Just use empty MetaData :) It's correct CanonicalProcess but not correct from Periodic POV

Reply:
Ok, so it was my misunderstanding of what it test does - reverted it